cves/kernel/CVE-2014-1690.yml

```diff
@@ -106,23 +107,23 @@ vcc_instructions: |
   Place any notes you would like to make in the notes field.
 vccs:
 - commit: 869f37d8e48f3911eb70f38a994feaa8f8380008
-  note: Discovered automatically by archeogit.
+  note: Discovered automatically by archeogit. Manually Confirmed.
 - commit: b20ab9cc63ca4605aec154cf54faa8455749f3f6
-  note: Discovered automatically by archeogit.
+  note: Discovered automatically by archeogit. Manually Confirmed.
 - commit: 3db05fea51cdb162cfa8f69e9cfb9e228919d2a9
```

Comment:
Check this and following VCC's again, the vulnerability may have already been introduced in an earlier commit.